assets/javascript/script.js

```diff
@@ -122,25 +123,24 @@ function wJax(googleOutput) {
     function displayWarning() {
       let advisoryDescription = $("<p>");
       let simpleAdvice = $("<p>");
-      let dangerIcon = $("<img/>");
-      dangerIcon.addClass('imgBox');
+      // let dangerIcon = $("<img/>");
```

Comment:
decide eventually if we want to keep

Reply:
Works for me! My most recent push included this in the code that I added, so we can change if needed. It currently works with the alert icons while running through multiple searches.